docker-for-azure/release-notes.md

```diff
@@ -26,12 +26,26 @@ title: Docker for Azure Release Notes
 
 ## Stable channel
 
+### 17.09 CE
+
+{{azure_blue_latest}}
+
+Release date: 10/6/2017
+
+- Docker Engine upgraded to [Docker 17.00.0 CE](https://github.com/docker/docker-ce/releases/tag/v17.09.0-ce){: target="_blank" class="_"}
```

Comment:
Same here

Reply:
Typo. I'll fix it. 0 and 9 are right next to each other.

Reply:
Fixed.